seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/catalog/AbstractJdbcCatalog.java

```diff
@@ -375,7 +371,11 @@ && listTables(tablePath.getDatabaseName())
     }
 
     @Override
-    public void createTable(TablePath tablePath, CatalogTable table, boolean ignoreIfExists)
+    public void createTable(
```

Comment:
Please do not change the existed interface. Why not create new method?
```java
void createTable(TablePath tablePath, CatalogTable table, boolean ignoreIfExists, boolean createIndex)
```

Reply:
> Please do not change the existed interface. Why not create new method?
> 
> ```java
> void createTable(TablePath tablePath, CatalogTable table, boolean ignoreIfExists, boolean createIndex)
> ```

Ok